plugins/GSdx/GSDeviceOGL.cpp

```diff
@@ -247,7 +247,7 @@ bool GSDeviceOGL::Create(GSWnd* wnd)
 	m_convert.cb = new GSUniformBufferOGL(g_convert_index, sizeof(ConvertConstantBuffer));
 	// Upload once and forget about it
 	ConvertConstantBuffer cb;
-	cb.ScalingFactor = GSVector4i(theApp.GetConfig("nativeres", 0) ? 1 : theApp.GetConfig("upscale_multiplier", 2));
+	cb.ScalingFactor = GSVector4i(theApp.GetConfig("upscale_multiplier", 1)==1 || theApp.GetConfig("upscale_multiplier", 1)==7 ? 1 : theApp.GetConfig("upscale_multiplier", 2));
```

Comment:
more information: https://github.com/PCSX2/pcsx2/blob/master/plugins/GSdx/GSdx.cpp#L298

https://msdn.microsoft.com/en-us/library/windows/desktop/ms724345%28v=vs.85%29.aspx

I have never encountered any problem in using them multiple times.